src/frontend/src/icp/components/transactions/IcTransactions.svelte

```diff
@@ -101,6 +101,9 @@
 
 	let noTransactions = false;
 	$: noTransactions = nonNullish($token) && $icTransactionsStore?.[$token.id] === null;
+
+	let hasIndexCanister = false;
+	$: hasIndexCanister = nonNullish($tokenAsIcToken) && nonNullish($tokenAsIcToken.indexCanisterId);
```

Comment:
Not sure that's a good idea but, maybe there is a Zod.safeParse available? But again not sure makes sense...

Reply:
Yes, there is `isIcTokenCanistersStrict` , I can use that, is that what you mean?

Reply:
Yes it's what I meant but, makes sense to you?

Reply:
So we keep it as it or you forgot pushing before pinging me for a new review?

No strong opinion, just asking.

Reply:
Yes yes, makes sense! Done!